### PR TITLE
PHPCS for assets and termmeta Utils

### PR DESCRIPTION
**Pull request by dlh01**

### CODE REVIEW

File: php/util/class-fieldmanager-util-term-meta.php [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
- * This is deprecated as of WordPress 4.4, which introduces Term Meta into core.
+ * @package Fieldmanager
+ */
+
+/**
+ * Use Fieldmanager to store meta data for taxonomy terms.
+ *
+ * This is deprecated as of WordPress 4.4, which introduces term MetA into core.
  * If you were using this feature prior to 4.4, it will continue to operate
  * until probably WordPress 4.8, however it is in your best interest to migrate
```
Comment (mslinnea) [inline php/util/class-fieldmanager-util-term-meta.php]: Curious, what's going to happen with WordPress 4.8 that would cause this to not work?

**Comment by mslinnea** [comment]
🚅

File: php/util/class-fieldmanager-util-term-meta.php (hunk 1 above)
Comment (dlh01) [inline php/util/class-fieldmanager-util-term-meta.php]: That is a better question for @mboynes, but my suspicion is that the comment means Fieldmanager probably will stop supporting `Fieldmanager_Util_Term_Meta` after 4.8.

Comment (mboynes) [inline php/util/class-fieldmanager-util-term-meta.php]: That's correct. That was just my estimation for when we'll officially deprecate it.

**Pull request merged**